ambry-api/src/main/java/com.github.ambry/config/StoreConfig.java

```diff
@@ -182,6 +182,23 @@
   @Default("115")
   public final int storeIndexPersistedEntryMinBytes;
 
+  /**
+   * Specifies the size threshold (as percentage of maximum size) of a store for converting the chunk to RO from RW
+   */
+  @Config(storeReadOnlyEnableSizeThresholdPercentageName)
+  @Default("95")
+  public final int storeReadOnlyEnableSizeThresholdPercentage;
+  public static final String storeReadOnlyEnableSizeThresholdPercentageName = "store.read.only.enable.size.threshold.percentage";
+
+  /**
+   * Specifies the size threshold delta below {@link #storeReadOnlyEnableSizeThresholdPercentageName} that a store will be
+   * converted from RO to RW
+   */
+  @Config(storeReadWriteEnableSizeThresholdPercentageDeltaName)
+  @Default("5")
+  public final int storeReadWriteEnableSizeThresholdPercentageDelta;
+  public static final String storeReadWriteEnableSizeThresholdPercentageDeltaName = "store.read.write.enable.size.threshold.percentage.delta";
```

Comment:
What is the reason to use a variable for this string?

Reply:
It's a long complicated string that, if not using a variable, is duplicated twice in the code (see line 240); easy to make a mistake between the two iterations and not catch it